mobile/src/src/components/Housing/Listings/ListingView.jsx

```diff
@@ -22,7 +22,7 @@ const ListingView = ({
   year,
   is_liked,
   likeListing,
-  createComment,
+  contentType,
```

Comment:
I wonder if this needs to be a prop? The listing view can only have the listing content type. This would simplify the renderItem() method in the ListingContainer